.github/workflows/upload-alerts.yml

```diff
@@ -0,0 +1,40 @@
+# upload alerts every 10 minutes
+
+name: Upload Alerts to AWS/Rockset
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
```

Comment:
Just FYI, GitHub doc says `The shortest interval you can run scheduled workflows is once every 5 minute`, but it doesn't seem to honor that in practice.  So it might not be run every 10m as you expect, it could be around every 20 to 30 I think

Reply:
I think it should be reasonable